coordinator/src/services/translation/response/cancellation/cancellation-medication-request.ts

```diff
@@ -31,32 +32,36 @@ export function createMedicationRequest(
       prescriptionStatusCode,
       prescriptionStatusDisplay,
       cancelRequesterPractitionerRoleId,
+      effectiveTime
     ),
     identifier: createItemNumberIdentifier(cancellationResponse.pertinentInformation1),
     status: medicationRequestStatus,
     intent: fhir.MedicationRequestIntent.ORDER,
     medicationCodeableConcept: MEDICINAL_PRODUCT_CODEABLE_CONCEPT,
     subject: fhir.createReference(patientId),
-    //TODO - effectiveTime should probably be the timestamp of the status, not authoredOn
-    authoredOn: convertHL7V3DateTimeToIsoDateTimeString(cancellationResponse.effectiveTime),
+    authoredOn: undefined,
```

Comment:
Do we really want this to be undefined? Should it even be in the model if we're not going to apply a value to it?

Reply:
i left it as undefined since i wanted it to be explicitly clear that the v3 message doesnt have enough information for authoredOn. this will never change so happy to delete the line. its in the model since the incoming FHIR medication request has authoredOn as a field

Reply:
OK, that makes sense. Just put a summarised version of that in the code as a comment. Presumably the serialiser will just remove the field altogether?